src/utils/azureUtils.ts

```diff
@@ -227,22 +223,40 @@ const supportedTypes: SupportedTypeMap = {
     'microsoft.web/serverfarms': { displayName: localize('serverFarms', 'App Service plans') },
     'microsoft.web/kubeenvironments': { displayName: localize('containerService', 'App Service Kubernetes Environment') },
     'microsoft.app/managedenvironments': { displayName: localize('containerAppsEnv', 'Container Apps Environment') },
-    'microsoft.app/containerapps': { displayName: localize('containerApp', 'Container Apps') },
+    'microsoft.app/containerapps': { displayName: localize('containerApp', 'Container Apps') }
```

Comment:
Dunno if this comma was removed intentionally but I like to leave them in arrays and objects, since in the future it will mean one less line changed when adding new rows.
```suggestion
    'microsoft.app/containerapps': { displayName: localize('containerApp', 'Container Apps') },
```

Reply:
I prefer trailing commas too, I think my formatter/ES Lint removed it. I'll have to try to turn that off